autoscale/src/main/java/com/sequenceiq/periscope/monitor/handler/ScalingRequest.java

```diff
@@ -143,6 +154,7 @@ private void scaleUp(int scalingAdjustment, int hostGroupNodeCount) {
         }
         String hostGroup = policy.getHostGroup();
         String statusReason = null;
+        FlowIdentifier flowIdentifier = new FlowIdentifier(FlowType.NOT_TRIGGERED, null);
```

Comment:
Replace with FlowIdentifier.notTriggered() - unless we want to explicitly use some kind of state to track triggers differently in Periscope.